File: code_examples/5-Holding_Data/15-For_Loop/CSharp/main.cs
/* Created by: Mr. Coxall
 * Created on: Sep 2020
 * This program uses a for loop
*/

using System;

/*
 * The Program class
*/
class Program {
    static void Main() {
        // this function uses a do ... while loop

        int positiveInteger;

        // input
        Console.Write("Enter how many times to repeat: ");
        positiveInteger = Convert.ToInt32(Console.ReadLine());
        Console.WriteLine();

        // process & output
        for (int loopCounter = 0; loopCounter < positiveInteger; loopCounter++) {
            Console.WriteLine(loopCounter + " time(s) through the loop.");
        }

        Console.WriteLine("\nDone.");
    }
}

File: code_examples/5-Holding_Data/15-For_Loop/CPP/main.cpp
// Copyright (c) 2020 St. Mother Teresa HS All rights reserved.
//
// Created by: Mr. Coxall
// Created on: Sep 2020
// This program uses a for loop

#include <iostream>

int main() {
    // this function uses a for loop
    int positiveInteger;

    // input
    std::cout << "Enter how many times to repeat: ";
    std::cin >> positiveInteger;
    std::cout << std::endl;

    // process & output
    for (int loopCounter = 0; loopCounter < positiveInteger; loopCounter++) {
        std::cout << loopCounter <<" time through loop." << std::endl;
    }

    std::cout << "\nDone." << std::endl;
    return 0;
}
